src/react/react.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Graffy from '@graffy/core';
+import { useQuery } from './';
+import { mockBackend } from '@graffy/testing';
+import { GraffyProvider } from './GraffyContext';
+import { renderHook } from '@testing-library/react-hooks';
+
+describe('useQuery', () => {
+  let g;
+  let wrapper;
+  let backend;
+
+  beforeEach(() => {
+    g = new Graffy();
+    backend = mockBackend();
+    backend.read = jest.fn(backend.read);
+    g.use(backend.middleware);
+    wrapper = ({ children }) => (
+      <GraffyProvider store={g}>{children}</GraffyProvider>
+    );
```

Comment:
Perhaps

```suggestion
    wrapper = function Wrapper ({ children }) {
      return (
        <GraffyProvider store={g}>{children}</GraffyProvider>
      );
    };
```

will fix the lint error?

Reply:
Throws `no-unused-vars` error.

To resolve this, can we please disable this rule?

Reply:
> To resolve this, can we please disable this rule?

Generally no.

If we legitimately need an unused variable in our code (but why would we?) we should add a `// eslint-disable-next-line no-unused-vars` comment and a separate comment explaining why this unused var is required.

Reply:
Alternately you can prefix unused variable names with an `_` (e.g. `_Wrapper`) to avoid the unused var error.